vendor/github.com/emicklei/go-restful/container.go

```diff
@@ -185,6 +185,11 @@ func logStackOnRecover(panicReason interface{}, httpWriter http.ResponseWriter)
 // when a ServiceError is returned during route selection. Default implementation
```

Comment:
this file is the only one that had changes that looked like they might impact us... I'd suggest a close review of the compression and handler type-checking